### PR TITLE
docs: update docs/Install.md

### PR DESCRIPTION
**Pull request by haraldh**
Fixes: #1805
Fixes: #1806
Fixes: #1807

Signed-off-by: Harald Hoyer <harald@profian.com>

### CODE REVIEW

**Review by MikeCamel — APPROVED** [review]

File: docs/Install.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-Enarx currently has support for Intel SGX and AMD SEV-SNP. Please check the [Requirements](#requirements).
+Enarx currently has support for Keeps running in Intel SGX and AMD SEV-SNP TEEs (Trusted Execution Environments). Please check the [Requirements](#requirements).  There is also support for application development using a "nil" backend which provides the same runtime, but no security isolation, and can run on a variety of Linux, Mac and Raspberry Pi systems.
```
Comment (rvolosatovs) [inline docs/Install.md:5]:
```suggestion
Enarx currently has support for Keeps running in Intel SGX and AMD SEV-SNP TEEs (Trusted Execution Environments). Please check the [Requirements](#requirements). There is also support for application development using a "nil" backend on a variety of Linux, Mac and Raspberry Pi systems, which provides the same runtime, but no security isolation.
```

File: docs/Install.md [hunk 2]
```diff
@@ -33,22 +34,11 @@ EOF
...
-- install an [SEV-SNP patched kernel](https://github.com/AMDESE/linux/tree/sev-snp-part2-v5)
-
-```sh:snp;ID=fedora
-$ sudo dnf copr enable -y harald/kernel-snp 
-$ sudo dnf install -y kernel{,-core,-modules}-5.14.0-0.rc2.28.sev.snp.part2.v5.fc34.x86_64
-```
+#### Hardware requirements for SEV
+For AMD our recommendation would be the EPYC 7003 Milan. This [article](https://www.servethehome.com/amd-epyc-7003-milan-sku-list-and-value-analysis/) offers an analysis of the different models.
 
+- You must run a Linux kernel with patches applied from this repo: https://github.com/AMDESE/linux/tree/sev-snp-part2-v6
```
[the remark this answers is not in the document]
Comment (haraldh) [reply docs/Install.md:41]: no, not yet

File: docs/Install.md (hunk 1 above)
Comment (MikeCamel) [reply docs/Install.md:5]: No.  I think the isolation is more important first.

File: docs/Install.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MikeCamel) [reply docs/Install.md:41]: Happy with the expansion, not the direct link.

File: docs/Install.md (hunk 1 above)
Comment (haraldh) [reply docs/Install.md:5]: me too

File: docs/Install.md [hunk 3]
```diff
@@ -14,14 +14,15 @@ Finally, in the [Running Enarx](#running-enarx) section, you'll build and run yo
...
-For Intel, our recommendation would be the 3rd Gen Intel Xeon Scalable Ice Lake. This [article](https://www.servethehome.com/3rd-gen-intel-xeon-scalable-ice-lake-sku-list-and-value-analysis/) provides a comprehensive analysis of the different models. The 5318Y or 5318S provide good value.
+### Setting up an SGX machine
+Enarx requires SGX2 (SGX with EDMM) support.
 
-For AMD our recommendation would be the EPYC 7003 Milan. This [article](https://www.servethehome.com/amd-epyc-7003-milan-sku-list-and-value-analysis/) offers an analysis of the different models. The 7313 seems like a good value.
+For Intel, our recommendation would be the 3rd Gen Intel Xeon Scalable Ice Lake. This [article](https://www.servethehome.com/3rd-gen-intel-xeon-scalable-ice-lake-sku-list-and-value-analysis/) provides a comprehensive analysis of the different models. The 5318Y or 5318S provide good value.
 
-### Setting up an SGX machine
-- Run a recent kernel with SGX support compiled in
+- You must run a Linux kernel with patches applied from this repo: https://github.com/enarx/linux/tree/sgx
```
[the remark this answers is not in the document]
Comment (haraldh) [reply docs/Install.md]: changed

**Pull request merged**